web_client/src/store/index.ts

```diff
@@ -231,13 +231,12 @@ function queueLoadScan(scan, loadNext = 0) {
         ];
       }
     } else {
-      let offset = 1;
+      let newIndex = scansInSameExperiment.indexOf(scan.id);
       while (!nextScan || !includeScan(nextScan.id)) {
+        if (newIndex >= scansInSameExperiment.length) break;
```

Comment:
I think it'd be more readable to just combine this into the loop condition, i.e.

```typescript
while ((!nextScan || !includeScan(nextScan.id)) && newIndex < scansInExperiment.length) {

Reply:
Changed in [dbd0ac5](https://github.com/OpenImaging/miqa/pull/627/commits/dbd0ac55bde23992de78bbeb686fea9661ca6217)